src/lang/de/fields.php

```diff
@@ -3,20 +3,20 @@
 return [
     'minutes' => [
         'every' => 'jede Minute',
-        'increment' => 'jede :increment Minute',
+        'increment' => 'alle :increment Minuten',
         'times_per_increment' => ':times mal alle :increment Minuten',
         'multiple' => ':times mal pro Stunde',
     ],
     'hours' => [
         'every' => 'jede Stunde',
-        'once_an_hour' => 'ein mal pro Stunde',
-        'increment' => 'jede :increment Stunde',
+        'once_an_hour' => 'einmal pro Stunde',
+        'increment' => 'alle :increment Stunden',
         'multiple_per_increment' => ':count Stunden aus :increment',
-        'times_per_increment' => ':times mal alle :increment Stunden',
+        'times_per_increment' => ':timesmal alle :increment Stunden',
```

Comment:
@lorisleiva technically this is correct. Does php translate the var correctly like this or is the space required for different reasons?

Reply:
Thanks for your help Mark! Yes, PHP is happy with this as it's just a simple find and replace for the substring `:variable`.